src/components/common/SeoHelmet.jsx

```
@@ -1,19 +1,24 @@
 import { Helmet } from 'react-helmet-async'
+const SeoHelmet = ({ title, description, image, url, noIndex = false }) => {
```

ESLint

[error] 'title' is missing in props validation

(react/prop-types)